.github/workflows/chromatic.yml

```diff
@@ -6,13 +6,15 @@ on:
   issue_comment:
     types: [created, edited]
 
+  push:
```

Comment:
uhm wait why would this build the baseline ah? 
this just says run this when something is pushed to dev, we are not doing smt like 'make dev as the baseline' since that is a state that is held within chromatic no?

Reply:
Hmm Chromatic has knowledge that the baseline is the `develop` branch, so every build on this branch would be updating this baseline. We would need to run this on every push to the `develop` branch so that we can get Chromatic to update the baseline they have stored, and PRs that trigger their respective Chromatic builds will be compared against this baseline.

This is known from the build here: https://www.chromatic.com/review?appId=61fcc8e14c3af9003a102ac9&number=1435&type=linked&view=changes. You can notice that there are no changes detected, even though that PR has introduced a new story. This is because previously we have been building branches as if they are `develop`, causing the baselines to keep being updated and thus we see no changes detected.